### PR TITLE
[Stream] Add changelog for live bandwidth estimates

### PR DESCRIPTION
**Pull request by irvinebroque**

### CODE REVIEW

**Review by dcpena — APPROVED** [review]

File: content/stream/changelog.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ weight: 11
 
 # Changelog
 
+## 2022-10-20
+
+### More accurate bandwidth estimates for live video playback
+
+When playing live video, Cloudflare Stream now provides significantly more accurate estimates of the bandwidth needs of each quality level to client video players. This ensures that live video plays at the highest quality that viewers have adequate bandwidth to play.
+
+As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in the HLS manifest.
```
Comment (nnazo) [inline content/stream/changelog.md]: > HLS manifest

This work applies for DASH manifests as well. I think we should we also mention it so folks don't get confused.

**Review by nnazo — CHANGES_REQUESTED** [review]

**Review by Everlag — APPROVED** [review]

**Comment by nnazo** [comment]
One comment on manifest phrasing, otherwise lgtm!

File: content/stream/changelog.md (hunk 1 above)
Comment (irvinebroque) [inline content/stream/changelog.md]:
```suggestion
As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in HLS (`.m3u8`) and DASH (`.mpd`) manifests.
```

**Review by nnazo — APPROVED** [review]
:+1:

File: content/stream/changelog.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ Cloudflare Stream now has no limit on the number of [live inputs](https://api.cl
 
 For more on creating and managing live inputs, get started with the [docs](/stream/stream-live/).
 
+## 2022-10-20
+
+### More accurate bandwidth estimates for live video playback
+
+When playing live video, Cloudflare Stream now provides significantly more accurate estimates of the bandwidth needs of each quality level to client video players. This ensures that live video plays at the highest quality that viewers have adequate bandwidth to play.
+
+As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide the most accurate estimate of the bandwidth requirements in HLS (`.m3u8`) and DASH (`.mpd`) manifests.
```
Comment (irvinebroque) [inline content/stream/changelog.md]:
```suggestion
As live video is streamed to Cloudflare, we transcode it to make it available to viewers at mulitple quality levels. During transcoding, we learn about the real bandwidth needs of each segment of video at each quality level, and use this to provide an estimate of the bandwidth requirements of each quality level the in HLS (`.m3u8`) and DASH (`.mpd`) manifests.
```

**Pull request merged**